### DISCUSSION TITLE
Homebridge running on docker no sambo-common

### DISCUSSION THREAD

**Discussion by AquaRelliux** [AlexGustafsson/homebridge-wol#223] [General]
Hey

I am running this on a unraid docker container. I am guessing samba is not added by default is there anyway to install this after the fact? I tried apt install but it cant find samba-common

**Comment by pat15312** [comment]
Try adding the following to the Docker Container startup.sh script:
`apt-get update && apt-get install -y samba-common`


You can also do this using the homebridge-config-ui-x plugin if you have it installed:

![image](https://user-images.githubusercontent.com/21065715/212784142-77f6a506-387f-489d-8a22-36b95c3d7e2c.png)
